tests/uricontent/uricontent-02/test.yaml

```diff
@@ -0,0 +1,26 @@
+args:
+- --set stream.midstream=true
+
+checks:
+- filter:
+    count: 2
+    match:
+      event_type: flow
+- filter:
+    count: 1
+    match:
+      event_type: stats
+- filter:
+    count: 2
+    match:
+      event_type: alert
+      alert.signature_id: 2
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 3
```

Comment:
Maybe for the second topic, maybe what Shivani suggested on a different PR could work: add a filter on pcap_cnt, since then we'd be able to check which packet is generating the specific alerts...